source/advanced_components/marketplace/market_linuxcontainers.rst

```diff
@@ -0,0 +1,47 @@
+.. _market_one:
+
+==============================
+Linuxcontainers MarketPlace
+==============================
+
+Overview
+================================================================================
+
+[Linux Containers image server](https://images.linuxcontainers.org/) hosts a public image server for use by LXC and LXD. It is the default image server on LXD.
+
+OpenNebula's linuxcontainers marketplace enables users to easily downlaod, contextualize and add Linux Container's images to an OpenNebula's image datastore. Linux Containers images are compressed **.tar.xz** files. In order to use them, this marketplace creates an image, where it dumps the content and later uploads it to OpenNebula.
```

Comment:
Mention also that Context packages are downloaded and added to the conditioner image